### PR TITLE
UpdatingProfile .md

### PR DESCRIPTION
**Pull request by Neethu-Mohan**
### Raw.Githack preview link

[Preview](https://raw.githack.com/Neethu-Mohan/Neethu-Mohan.github.io/Profile/pages/vi/profiles/Neethu-Mohan.md)

### CODE REVIEW

**Review by Nida8930 — CHANGES_REQUESTED** [review]
You're on the right track! I liked that you used a table. 

In your raw.githack link you need to add `#!` after Profile/ and before pages, so it links to a preview and not raw markdown. 
Also on your page it does not appear that your .jpg is loading correctly.

**Comment by jamjakpa** [comment]
https://raw.githack.com/Neethu-Mohan/Neethu-Mohan.github.io/Profile/<code>somethinghastogoinhere</code>/pages/vi/profiles/Neethu-Mohan.md

**Comment by Neethu-Mohan** [comment]
Please check https://neethu-mohan.github.io/#!index.md and https://raw.githack.com/Neethu-Mohan/Neethu-Mohan.github.io/Profile/#!pages/vi/profiles/Neethu-Mohan.md

**Review by jamjakpa — APPROVED** [review]

**Comment by Nida8930** [comment]
> https://raw.githack.com/Neethu-Mohan/Neethu-Mohan.github.io/Profile/`somethinghastogoinhere`pages/vi/profiles/Neethu-Mohan.md

Just one `/` between Profile and pages.

**Review by Nida8930 — APPROVED** [review]
Looks good, good job.

File: pages/vi/profiles/Neethu-Mohan.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Neethu Mohan
+  * * *
+## About me
+I am originally from India and currently living in Abudhabi, UAE. I have a master degree in Wireless Networks and Applications and a bachelor degree in Electrical and Electronics Engineering. I was working as a server Engineer. I am certified in both RHCSA and RHCE. I am good in Python and bash scripting. I am also familiar with C and C++.
+
+![](https://raw.githubusercontent.com/Neethu-Mohan/Neethu-Mohan.github.io/Profile/pages/vi/profiles/images/MyCity.jpg)
```
Comment (dogi) [inline pages/vi/profiles/Neethu-Mohan.md:6]: this should be a relative link

**Pull request merged**